roles/manifest/README.md

```diff
@@ -0,0 +1,51 @@
+theforeman.foreman.manifest
+===========================
+
+Upload Subscription Manifest
+
+Role Variables
+--------------
+
+This role supports the [Common Role Variables](https://github.com/theforeman/foreman-ansible-modules/blob/develop/README.md#common-role-variables).
+
+- `manifest_path`: Where to find the Manifest on Foreman server. When using manifest_download, it is first downloaded to this location.
+- `manifest_download`: Whether to first download the Manifest from the Red Hat Customer Portal.
```

Comment:
Given the user needs to provide the UUID, isn't that being present enough of a marker that a download should happen?

Reply:
Yes, internally. I think the price is that the behavior is less obvious to the user.